Firestore/core/test/firebase/firestore/model/mutation_test.cc

```diff
@@ -31,14 +31,15 @@ using testutil::Doc;
 using testutil::SetMutation;
 
 TEST(Mutation, AppliesSetsToDocuments) {
-  Document base_doc = Doc("collection/key", 0,
-                          {{"foo", FieldValue::FromString("foo-value")},
-                           {"baz", FieldValue::FromString("baz-value")}});
+  std::shared_ptr<Document> base_doc = std::make_shared<Document>(
```

Comment:
Optional: consider using `auto` here to avoid spelling out the type twice (`make_shared` makes the type clear).

Reply:
Already done (prior to your review). I think you're looking at only a subset of the changes.